test/unit/assorted/server_selection_latency_window_utils.ts

```diff
@@ -145,4 +145,6 @@ export async function runServerSelectionLatencyWindowTest(test: ServerSelectionL
   const observedFrequencies = calculateObservedFrequencies(selectedServers);
 
   compareResultsToExpected(test.outcome, observedFrequencies);
+
+  await topology.close();
```

Comment:
This is one of the causes of leaks in unit tests - gotta close the topology.

Before the change in this PR, the pool did not run the background thread by default.  With this change, the background thread does run by default, which keeps the event loop open unless the pool's minPoolSize timer is explicitly cleared (which happens in pool.close()).

This isn't an issue for in real scenarios because the server class always closes its pool.